FAQ.md

```diff
@@ -341,3 +347,11 @@ Note this option is not shown in the output of `etcd --help`.
 ```
 
 Indeed this distinction was eliminated by etcd in their master branch, but not backport to announced versions, so be care when deploy your etcd cluster.
+
+## How to achieve high availability of APISIX?
```

Comment:
What about "How to set up high available Apache APISIX clusters"

Reply:
updated.